src/components/TextField/TextField.jsx

```diff
@@ -104,6 +105,12 @@ const TextField = forwardRef(
       elementType: "div"
     });
 
+    useEffect(() => {
+      if (inputRef.current && autoFocus) {
```

Comment:
add `cancelAnimationFrame` on the return function of the `useEffect` please

Reply:
@orrgottlieb found that i missed combo aswell please look there too :)